src/utils/amplifyServerUtils.ts

```diff
@@ -0,0 +1,27 @@
+import { createServerRunner } from '@aws-amplify/adapter-nextjs';
+const OAUTH_DOMAIN: string = process.env.NEXT_PUBLIC_OAUTH_DOMAIN || '';
+const LOCALHOST_URL: string = process.env.NEXT_PUBLIC_LOCALHOST_URL || '';
+const OAUTH_REDIRECT_URL: string = process.env.NEXT_PUBLIC_OAUTH_REDIRECT_URL || '';
+const USER_POOL_ID: string = process.env.NEXT_PUBLIC_USER_POOL_ID || '';
+const USER_POOL_CLIENT_ID: string = process.env.NEXT_PUBLIC_USER_POOL_CLIENT_ID || '';
+
+// eslint-disable-next-line @typescript-eslint/no-unsafe-call
```

Comment:
https://github.com/aws-amplify/amplify-js/blob/04627618afe526dd776ee678b723ed6e697f440d/packages/adapter-nextjs/src/types/NextServer.ts#L89-L91

This method's type seems not to be open for external so I cannot help setting this ignore of eslint check for now.

https://docs.amplify.aws/nextjs/build-a-backend/server-side-rendering/nextjs-v5-to-v6-migration-guide/#using-the-api-category-in-v6